pkg/resource/clickpipe.go

```diff
@@ -567,6 +568,189 @@ func (c *ClickPipeResource) Schema(_ context.Context, _ resource.SchemaRequest,
 							},
 						},
 					},
+					"postgres": schema.SingleNestedAttribute{
+						MarkdownDescription: "The Postgres CDC source configuration for the ClickPipe.",
+						Optional:            true,
+						Attributes: map[string]schema.Attribute{
+							"host": schema.StringAttribute{
+								Description: "The hostname of the Postgres instance.",
+								Required:    true,
+								PlanModifiers: []planmodifier.String{
+									stringplanmodifier.RequiresReplace(),
```

Comment:
OpenAPI allows host, port and database to be updated in-place. Host, for example, is useful when switching from direct connection to private link